### PR TITLE
feat(fhir-converter): added date mapping to med req

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport-internal#2128

### Description
- Added `authoredOn` field mapping for the `MedicationRequest` resource

### Testing

- Local
  - [x] Ran the FHIR test suite without insertion. 
  - [x] FHIR insert on a subfolder of the FHIR test suite - 91 files processed, no errors. 

### Release Plan
- [ ] Merge this

### CODE REVIEW

File: packages/fhir-converter/src/templates/cda/Resources/MedicationRequest.hbs [hunk 1]
```diff
@@ -37,6 +37,13 @@
         ],
         "intent":"order",
         "status":{{>ValueSet/MedicationRequestStatus.hbs code=medicationRequest.statusCode.code}},
+        {{#if medicationRequest.effectiveTime.high.value}}
+            "authoredOn": "{{formatAsDateTime medicationRequest.effectiveTime.high.value}}",
+        {{else if medicationRequest.effectiveTime.low.value}}
+            "authoredOn": "{{formatAsDateTime medicationRequest.effectiveTime.low.value}}",
+        {{else if medicationRequest.author.time}}
+            "authoredOn": "{{formatAsDateTime medicationRequest.author.time}}",
+        {{/if}}
```
Comment (jonahkaye) [inline packages/fhir-converter/src/templates/cda/Resources/MedicationRequest.hbs:46]: what about dosage? I am pretty sure we can get that too right?

**Review by jonahkaye — APPROVED** [review]
Approved pending fhir insert

File: packages/fhir-converter/src/templates/cda/Resources/MedicationRequest.hbs (hunk 1 above)
Comment (RamilGaripov) [reply packages/fhir-converter/src/templates/cda/Resources/MedicationRequest.hbs:46]: Perhaps, a separate issue to fix at a later time. I'll create a ticket

**Review by jonahkaye — CHANGES_REQUESTED** [review]
add this
https://github.com/metriport/metriport/pull/1916/files

**Review by jonahkaye — APPROVED** [review]

**Pull request merged**